examples/pid_lower_than_1000.yaml

```diff
@@ -0,0 +1,13 @@
+name: pid_lower_than_1000
+description: This policy traces read and write syscalls for pids lower than 1000
```

Comment:
actually pids can wrap, so better use equality (`pid=1000`) instead of inequality in this example